griptape/tools/web_search/tool.py

```diff
@@ -15,25 +15,30 @@
 
 @define
 class WebSearch(BaseTool):
-    web_search_driver: BaseWebSearchDriver = field(default=None, kw_only=True)
+    web_search_driver: BaseWebSearchDriver = field(kw_only=True)
 
     @activity(
         config={
-            "description": "Can be used for searching the web",
+            "description": "Can be used for searching the web via the {{ _self.web_search_driver.__class__.__name__}}.",
```

Comment:
Might be nice if the web search drivers had a method that returns the name of search engine. I assume that LLMs might like it a little better than this class name.

Reply:
Yeah I was thinking how far to take this. A non-Driver name is probably better.

Reply:
strip out the `WebSearchDriver` part of the name?